tests/test_indefinite_freeze_attack.py

```diff
@@ -15,6 +15,14 @@
     than verifying text output), use pre-generated repository files, and
     discontinue use of the old repository tools. -vladimir.v.diaz
 
+  March 9, 2016.
+    Additional test added relating to issue:
+    https://github.com/theupdateframework/tuf/issues/322
+    If a metadata file is not downloaded (no indication of a new version
```

Comment:
I'm not sure if this should instead say, "... a metadata file is not updated ..." or "... the expiration of the locally trusted metadata should be detected."